src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,32 @@
+$card-width: 200px;
+$card-height: 406px;
+
+$card-container-margin: 50px;
+$card-container-width: (4 * $card-width + 3 * $card-container-margin);
+
+$button-height: 40px;
+$button-width: 166px;
+$stars-size: 16px;
+
+$main-font-family: "Roboto", sans-serif, arial;
```

Comment:
it is not advisable to use variable for font family , font size as you can put them for html or body and those will inherit by child and grand child elements